.github/workflows/spack_default_build.yaml

```diff
@@ -101,6 +101,7 @@ jobs:
         spack_spec:
           - gridkit@develop +enzyme
             ^enzyme@0.0.173
+            ^sundials@develop
```

Comment:
This is fine for now, but as soon as the next SUNDIALS version is released, we should use that version. Breaking changes can occur on the develop branch.